docs/v2/spec/2018_Modified_LAR_Data_Dictionary.csv

```diff
@@ -0,0 +1,86 @@
+Ethnicity of Co-Applicant or Co-Borrower: 2,Character Varying,1 11 12 13 14 2 blank
+Ethnicity of Co-Applicant or Co-Borrower: 3,Character Varying,1 11 12 13 14 2 blank
+Ethnicity of Co-Applicant or Co-Borrower: 4,Character Varying,1 11 12 13 14 2 blank
+Ethnicity of Co-Applicant or Co-Borrower: 5,Character Varying,1 11 12 13 14 2 blank
+Ethnicity of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3
+Ethnicity of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3 4
+Race of Applicant or Borrower: 1,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 6 7 blank
+Race of Applicant or Borrower: 2,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Applicant or Borrower: 3,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Applicant or Borrower: 4,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Applicant or Borrower: 5,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Co-Applicant or Co-Borrower: 1,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 6 7 8 blank
+Race of Co-Applicant or Co-Borrower: 2,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Co-Applicant or Co-Borrower: 3,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Co-Applicant or Co-Borrower: 4,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Co-Applicant or Co-Borrower: 5,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 blank
+Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3
+Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3 4
+Sex of Applicant or Borrower,Numeric,1 2 3 4 6
+Sex of Co-Applicant or Co-Borrower,Numeric,1 2 3 4 5 6
+Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3
+Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Numeric,1 2 3 4
+Age of Applicant or Borrower,Character Varying,<25; 25-34; 35-44; 45-54; 55-64; 65-74; >74
+Age of Applicant >= 62,Character Varying,Yes; No; NA
+Age of Co-Applicant or Co-Borrower,Character Varying,<25; 25-34; 35-44; 45-54; 55-64; 65-74; >74
+Age of Co-Applicant >= 62,Character Varying,Yes; No; NA
+Income,Character Varying,Integers or 'NA'
+Type of Purchaser,Numeric,0 1 2 3 4 5 6 71 72 8 9
+Rate Spread,Character Varying Doubles (Double-precision floating point. For example:  3.25),'NA' or 'Exempt'
+HOEPA Status,Numeric,1 2 3
+Lien Status,Numeric,1 2
+Applicant or Borrower - Name and Version of Credit Scoring Model,Numeric,1 2 3 4 5 6 7 8 9 1111
+Co-Applicant or Co-Borrower - Name and Version of Credit Scoring Model,Numeric,1 2 3 4 5 6 7 8 9 10 1111
+Reason for Denial: 1,Numeric,1 2 3 4 5 6 7 8 9 10 1111

```

Comment:
In denial reason 2-4 the data type should be numeric. The blank in the FIG will populate as a null value when loaded to a database. An empty string is a different value than a blank in the .txt files we receive in submission.

Reply:
Sure. This is reflected in the latest commit.